### DISCUSSION TITLE
How to add and remove people from Github organisations

### DISCUSSION THREAD

**Discussion by Cross-repo** [community/community#114165] [Enterprise] [answered]
Please can someone tell me how to add or remove contributors to my github organisation? I can not do it through "People".

This is incredibly frustrating for me and my team.

Please help!

Thank you.

**Comment by Tgentil** [comment, marked as answer]
Adding or removing contributors to a GitHub organization should be straightforward through the "People" section of the organization's settings, but it sounds like you're encountering some issues. Here's a basic guide to help you:

To Add People:

1. Navigate to your organization's GitHub page.
2. Click on "People" in the organization's settings.
3. Use the "Invite member" button to invite people by typing their username.


To Remove People:

1. Again, go to "People" in the organization's settings.
2. Find the member you want to remove, and you should see options to remove or change their role next to their name.

> Remember, you need to be an owner of the organization to manage members effectively. If you are a member with limited privileges, you might not be able to add or remove contributors. Also, organizational rules might affect your ability to make these changes.

If you're still having trouble, it might help to see a screenshot of what you're seeing. This can provide more context to diagnose the problem.

This is a picture of my organization that may help you out:

![My organization](https://github.com/community/community/assets/88681055/993359e0-e757-4152-8bc0-edd8009f1da7)


For more detailed instructions and guidelines, you can refer to GitHub's official documentation on [adding people to your organization](https://docs.github.com/articles/adding-people-to-your-organization) and [about organization membership](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/about-organization-membership).

Hope this helps!

**Comment by queenofcorgis** [comment]
@Cross-repo did @Tgentil's response help or are you still in need of assistance?
